projects/angular/src/forms/range/STYLES.md

```diff
@@ -0,0 +1,17 @@
+# Range
+
+## CSS Custom Properties
+
+| CSS Propertie                         | Description                           |
+| ------------------------------------- | ------------------------------------- |
+| --clr-forms-range-progress-fill-color | The color of the range fill progress. |
+| --clr-forms-range-track-color         | The color of the range track.         |
+
+## Classes
+
+| Class Name          | Description                                                 |
+| ------------------- | ----------------------------------------------------------- |
+| .clr-range-wrapper  | Represents the wrapper element for the range input.         |
+| .fill-input         | Represents the progress bar element inside the range input. |
+| .clr-range          | Represents the range input element.                         |
+| input[type='range'] | Represents the range input element.                         |
```

Comment:
This is technically not a class. Though we use it to override basic design. But the users should be able to identify it without our help as it's pure html selector.
Let's remove it.